provisionerd/provisionerd.go

```diff
@@ -261,6 +282,18 @@ func (p *Server) acquireJob(ctx context.Context) {
 		return
 	}
 
+	// This prevents loads of provisioner daemons from consistently sending
+	// requests when no jobs are available.
+	//
+	// The debounce only occurs when no job is returned, so if loads of jobs are
+	// added at once, they will start after at most this duration.
+	lastAcquireMutex.RLock()
+	if !lastAcquire.IsZero() && time.Since(lastAcquire) < p.opts.JobPollDebounce {
+		lastAcquireMutex.RUnlock()
+		return
+	}
+	lastAcquireMutex.RUnlock()
```

Comment:
Should disable this debounce if running in tests

Reply:
It's only enabled in `coder server`